contribute/general/gokit.md

```diff
@@ -1,40 +1,40 @@
+    While the packages in the kit project are categorised as per the functionality, the naming convention should ideally provide the reader with the information on what the package “provides”. So, the packages (in a kit project) must provide, not contain. In Maya, we have packages like types, orchprovider, volumes etc. name of these packages suggests the functionality provided by them.
+
 - Portability
     
-    Portability is important factor for packages in kit project. Hence, we are making maya in such a way that it will be easy to import and use in any Go project. Packages in the Maya are not single point of dependency and all the packages are independent of each other. For example, types directory contains versioned Kubernetes and Nomad packages. These packages are simply importable to any project to use kubernetes and Nomad API’s.
+    Portability is an important factor for packages in a kit project. Hence, we are making Maya in such a way that it will be easy to import and use in any Go project. Packages in Maya are not a single point of dependency and all the packages are independent of each other. For example, types directory contains versioned Kubernetes and Nomad packages. These packages are simply importable to any project to use Kubernetes and Nomad API’s.
 
-## Example usage of maya kit project
-Maya-apiserver uses maya as a Kit project. Maya-apiserver exposes OpenEBS operations in form of REST APIs. This allows multiple clients e.g. volume related plugins to consume OpenEBS storage operations exposed by Maya API server. Maya-apiserver will use volume provisioner as well as orchestration provider modules from Maya. Maya-apiserver will always have HTTP endpoints to do OpenEBS operations.
-Similarly, openebs-k8s-provisioner will use the maya-kit project kubernetes API to query for details about the Storage Classes, etc.,
+## Example usage of Maya kit project
+Maya-apiserver uses Maya as a kit project. Maya-apiserver exposes OpenEBS operations in form of REST APIs. This allows multiple clients e.g. volume related plugins to consume OpenEBS storage operations exposed by Maya API server. Maya-apiserver will use volume provisioner as well as orchestration provider modules from Maya. Maya-apiserver will always have HTTP endpoints to do OpenEBS operations.
+Similarly, openebs-k8s-provisioner will use the maya-kit project Kubernetes API to query for details about the storage classes, etc.
 Another usage is of the maya-kit project, maya-apiserver client that is accessed by maya-cli as well as the openebs-k8s-provisioner to talk to maya-apiserver.
 
 ## Conclusion
-Go Kit project should contain packages which are usable, purposeful and portable. Go Kit projects will improve the efficiency of the organization at both human and code level.
+Go kit project should contain packages which are usable, purposeful and portable. Go Kit projects will improve the efficiency of the organization at both human and code level.
```

Comment:
GO

Reply:
I think `Go` is the right way to go. :smile: 
I haven't changed that anywhere.

Reply:
Yes. Go is the right way to GO :-)

Reply:
@kmova Thanks for showing the right way to go. The PR #1542 might also be ready to GO.
***_This marks the end of GO puns._***